src/pages/SequenceEdit/SequenceEdit.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { Card, IconInfo16, NoticeBox } from '@dhis2/ui'
+import { useParams } from 'react-router-dom'
+import i18n from '@dhis2/d2-i18n'
+import { Spinner } from '../../components/Spinner'
+import { SequenceEditFormContainer } from '../../components/Forms'
+import { useJobScheduleById } from '../../hooks/job-schedules'
+import styles from './SequenceEdit.module.css'
+
+const SequenceEdit = () => {
+    const { id } = useParams()
+    const { data, fetching, error } = useJobScheduleById(id)
+
+    if (fetching) {
+        return <Spinner />
+    }
+
+    if (error) {
+        return (
+            <NoticeBox
+                error
+                title={i18n.t('Could not load requested schedule')}
```

Comment:
Do you want to refer to these as schedule here? Or sequence? (Also applies in next translation string)

Reply:
Yeah I think that this should be called a queue actually. I've created a follow up issue for it.